submissions/chabuChabu/php/LoopsAndNumber.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+//Display every number between 100 and 150
+
+for($i = 101;$i < 150;$i++){
```

Comment:
Well done @ChabuChabu. Your for loop argument is hard to read. Try adding some spaces in between.

Reply:
@TundeOladejo  for loop made more readable